src/internal/sdata/sdata_test.go

```diff
@@ -224,8 +184,24 @@ func TestSQLTupleWriter(suite *testing.T) {
 				// for mysql compatibility
 				*ti = time.Now()
 			})
+			fz.Funcs(func(x *interface{}, co fuzz.Continue) {
```

Comment:
Hmm, this seems like a duplicate of https://github.com/pachyderm/pachyderm/pull/7637/files#diff-16ac2fdb87cc8640c10a72c7be681809939ca82348c89981bef0b2de48477645R115

I feel like we can refactor `GenerateTestData` to just generate data and not execute the SQL. Then we can re-use that logic where is needed.